test/DualTest.jl

```diff
@@ -444,24 +444,23 @@ for N in (0,3), M in (0,4), V in (Int, Float32)
                 end
             elseif arity == 2
                 derivs = DiffRules.diffrule(M, f, :x, :y)
+                x, y = if f === :ldexp
+                    rand(), rand(1:10)
+                elseif f === :mod
+                    13 + rand(), 5 + rand() # make sure x/y is not an integer
```

Comment:
Random values where `x/y` where (close to) an integer caused problems in DiffRules (https://github.com/JuliaDiff/DiffRules.jl/pull/78), so I thought it's good to apply the same fix here as well.